test/hotspot/jtreg/runtime/ClassFile/OOMCrashClass4000_1.java

```diff
@@ -0,0 +1,111 @@
+import org.objectweb.asm.AnnotationVisitor;
```

Comment:
This needs the copyright header.  See the one in the java file for an example.  The copyright should be just 2025 for this file since it's new.  The copyright for the other file should be changed to include 2025 as the end date.  Note the strange looking trailing comments.